poptorch_experimental_addons/__init__.py

```diff
@@ -24,3 +24,4 @@ def _load_native_library() -> None:
 _load_native_library()
 
 from . import sparse  # NOQA:F401,E402
+from . import collectives
```

Comment:
Perhaps `from . import collectives, sparse` on the line above (and then you'll get to share the NOQA).